src/ExplorerRoutes.tsx

```diff
@@ -31,11 +31,7 @@ export default function ExplorerRoutes() {
         </Route>
         <Route path="/account">
           <Route
-            path=":address/modules/:selectedModuleName/:modulesTab"
-            element={<AccountPage />}
-          />
-          <Route
-            path=":address/modules/:selectedModuleName"
+            path=":address/modules/:modulesTab/:selectedModuleName"
```

Comment:
makes sense to me, tho it could be a breaking change if other people used old urls to do something. but i think effect should be minimal.